gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ownership/acceptOwnership.ts

```diff
@@ -31,18 +32,19 @@ export const makeAcceptOwnershipCommand = (
       return this
     }
 
-    makeRawTransaction = async (signer: PublicKey) => {
+    makeRawTransaction = async () => {
       const contract = getContract(contractId, '')
       const address = contract.programId.toString()
       const program = this.loadProgram(contract.idl, address)
+      const authority = new PublicKey(this.flags.authority)
 
       const state = new PublicKey(this.args[0])
 
       const tx = await program.methods
         .acceptOwnership()
         .accounts({
           state: state,
-          authority: signer,
+          authority: authority,
```

Comment:
This should be sent as the signer from every environment, including multisig. Let's wait to see how the issue resolves